app/script/event/WebApp.js

```diff
@@ -36,6 +36,22 @@ z.event.WebApp = {
     PLAY_IN_LOOP: 'wire.webapp.audio.play_in_loop',
     STOP: 'wire.webapp.audio.stop',
   },
+  BACKUP: {
```

Comment:
@atomrc Will we still need all of these events even if we go without Wrapper <> Webapp communication?

Reply:
No you're right, I believe we don't. 
We might need a single event though, for the inter-viewmodel communication to plug the UI @AndyLnd is building. But since this is on another branch, I believe it's better we remove it there and re-add the one we need on the other branch

Reply:
Ok, History Backup is merged. Let's revisit the events.

Reply:
These are legit events needed for the UI (@AndyLnd needed those).